internal/tools/aws/route53.go

```diff
@@ -133,11 +133,6 @@ func (a *Client) deleteCNAME(hostedZoneID, dnsName string, logger log.FieldLogge
 		return err
 	}
 
-	if len(dnsName) >= 64 {
```

Comment:
what happen if the deletion of the route53 fail? should we retry? should we ignore, log and continue?

cc @stylianosrigas @gigawhitlocks @gabrieljackson

Reply:
From the perspective of this helper routine we should just return `err` like we do on line 179. As for the behavior of the broader deletion routine I have less strong opinions about retrying.

Reply:
Yeah, it should fail if there is a record and it was not deleted. The logic in this function should do that for us, but will skip everything if there is no record: https://github.com/mattermost/mattermost-cloud/pull/139/files#diff-41cf3d82a35fe36c66189425b60f6473R169